File: examples/python/basic.py
import requests
import json

def verify_page():
    url = "https://proof.reprompt.jp/api/browser-proof"
    payload = {
        "url": "https://example.com",
        "task": "Check if this domain is for use in documentation."
    }
    headers = {
        "Content-Type": "application/json"
    }

    response = requests.post(url, json=payload, headers=headers)
    
    if response.status_code == 200:
        data = response.json()
        print(f"Success: {data['success']}")
        print(f"Answer: {data['answer']}")
        print(f"Evidence: {data['evidence_text']}")
    else:
        print(f"Error: {response.status_code}")
        print(response.text)

if __name__ == "__main__":
    verify_page()

File: examples/js/basic.ts
async function verifyPage() {
  const BROWSER_PROOF_BASE_URL = "https://proof.reprompt.jp";
  
  const response = await fetch(`${BROWSER_PROOF_BASE_URL}/api/browser-proof`, {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
      "X-API-Key": "YOUR_API_KEY"
    },
    body: JSON.stringify({
      url: "https://example.com",
      task: "Verify that this domain is used for documentation examples."
    })
  });
  
  const result = await response.json();
  console.log("Verification Success:", result.success);
  console.log("Evidence:", result.evidence_text);
  console.log("Screenshot:", result.screenshot_url);
}

verifyPage();
